<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextEdit" name="textEingabe">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>601</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>601</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="sendButton">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>101</y>
     <width>601</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>senden</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
